homeassistant/components/squeezebox/sensor.py

```diff
@@ -0,0 +1,110 @@
+"""Platform for sensor integration for squeezebox."""
+
+from __future__ import annotations
+
+import logging
+from typing import cast
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.const import EntityCategory, UnitOfTime
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+
+from . import SqueezeboxConfigEntry
+from .const import (
+    STATUS_SENSOR_INFO_TOTAL_ALBUMS,
+    STATUS_SENSOR_INFO_TOTAL_ARTISTS,
+    STATUS_SENSOR_INFO_TOTAL_DURATION,
+    STATUS_SENSOR_INFO_TOTAL_GENRES,
+    STATUS_SENSOR_INFO_TOTAL_SONGS,
+    STATUS_SENSOR_LASTSCAN,
+    STATUS_SENSOR_NEWPLUGINS,
+    STATUS_SENSOR_NEWVERSION,
+    STATUS_SENSOR_OTHER_PLAYER_COUNT,
+    STATUS_SENSOR_PLAYER_COUNT,
+)
+from .entity import LMSStatusEntity
+
+SENSORS: tuple[SensorEntityDescription, ...] = (
+    SensorEntityDescription(
+        key=STATUS_SENSOR_INFO_TOTAL_ALBUMS,
+        state_class=SensorStateClass.TOTAL,
+        icon="mdi:album",
+    ),
+    SensorEntityDescription(
+        key=STATUS_SENSOR_INFO_TOTAL_ARTISTS,
+        state_class=SensorStateClass.TOTAL,
+        icon="mdi:account-music",
+    ),
+    SensorEntityDescription(
+        key=STATUS_SENSOR_INFO_TOTAL_DURATION,
+        state_class=SensorStateClass.TOTAL,
+        device_class=SensorDeviceClass.DURATION,
+        native_unit_of_measurement=UnitOfTime.SECONDS,
+    ),
+    SensorEntityDescription(
+        key=STATUS_SENSOR_INFO_TOTAL_GENRES,
+        state_class=SensorStateClass.TOTAL,
+        icon="mdi:drama-masks",
+    ),
+    SensorEntityDescription(
+        key=STATUS_SENSOR_INFO_TOTAL_SONGS,
+        state_class=SensorStateClass.TOTAL,
+        icon="mdi:file-music",
+    ),
+    SensorEntityDescription(
+        key=STATUS_SENSOR_LASTSCAN,
+        device_class=SensorDeviceClass.TIMESTAMP,
+    ),
+    SensorEntityDescription(
+        key=STATUS_SENSOR_NEWPLUGINS,
+        entity_registry_visible_default=False,
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
+    SensorEntityDescription(
+        key=STATUS_SENSOR_NEWVERSION,
+        entity_registry_visible_default=False,
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
+    SensorEntityDescription(
+        key=STATUS_SENSOR_PLAYER_COUNT,
+        state_class=SensorStateClass.TOTAL,
+        icon="mdi:folder-play",
+    ),
+    SensorEntityDescription(
+        key=STATUS_SENSOR_OTHER_PLAYER_COUNT,
+        state_class=SensorStateClass.TOTAL,
+        entity_registry_visible_default=False,
+        icon="mdi:folder-play-outline",
+    ),
```

Comment:
What does this return

Reply:
But don't we see all the players as devices?

Reply:
Total Players yes, Other Players no (other servers), You thinking Diagnostic? we get them for free as part of the status poll.

Reply:
And other servers are added to this number?

Reply:
no total players is for this server/service

Reply:
I have tried to rename this sensor a bit more sensibly also.